### PR TITLE
IacType: relocate from cloudrail repo

### PR DESCRIPTION
**Pull request by idan-aviv**

### CODE REVIEW

**Review by Imanuel-Mizrahi — APPROVED** [review]

File: cloudrail/knowledge/context/mergeable.py [hunk 1]
```diff
@@ -82,13 +84,18 @@ def origin(self) -> EntityOrigin:
         if self.is_pseudo:
             return EntityOrigin.PSEUDO
         elif self.iac_state:
-            return EntityOrigin.TERRAFORM
+            if self.iac_state.iac_type == IacType.TERRAFORM:
+                return EntityOrigin.TERRAFORM
+            elif self.iac_state.iac_type == IacType.CLOUDFORMATION:
+                return EntityOrigin.CLOUDFORMATION
+            else:
+                return EntityOrigin.NONE
```
Comment (yi2020) [inline cloudrail/knowledge/context/mergeable.py:92]: Do we know what to do with NONE? Wouldn't it make more sense to throw an exception here?

Comment (Tomerva) [reply cloudrail/knowledge/context/mergeable.py:97]: i agree, None makes no sense and if we get None here for some reason we should raise an exception

**Review by idan-aviv — COMMENTED** [review]
@Tomerva @yi2020 done

File: cloudrail/knowledge/context/mergeable.py (hunk 1 above)
Comment (idan-aviv) [reply cloudrail/knowledge/context/mergeable.py]: @yi2020 @Tomerva done

**Comment by idan-aviv** [comment]
duplicated

**Pull request closed without merging**